src/tests/multihost/alltests/test_rfc2307bis.py

```diff
@@ -0,0 +1,62 @@
+""" Automation of rfc2307bis
+
+:requirement: rfc2307bis
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+:status: approved
+"""
+import os
+import re
+import posixpath
+import pytest
+from sssd.testlib.common.utils import sssdTools
+from constants import ds_instance_name
+
+@pytest.mark.usefixtures('setup_sssd')
+@pytest.mark.rfc2307bis
+class Testrfc2307bis(object):
+    """
+    This is test case class for ldap rfc2307bis
+        :setup:
+          1. Configure SSSD to authenticate against directory server
+          2. Enable debug_level to 9 in the 'nss', 'pam' and domain section
+    """
+    @staticmethod
+    @pytest.mark.tier1_3
```

Comment:
tire1_3 is full , now you should add tests on tire1_4

Reply:
Done